### ISSUE TITLE
Documentation bug in 'Generating Deferreds'

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3524]
|<img alt="eggy's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| eggy reported|
|-|-|
|Trac ID|trac#3524|
|Type|defect|
|Created|2008-11-05 20:17:38Z|

In section 'Integrating blocking code with Twisted' of http://twistedmatrix.com/projects/core/documentation/howto/gendefer.html a comment in the code in function 'run' says: 
# unless the largeFibonnaciNumber thread returns very fast, these print
#lines should happen first
They always happen first, it's not possible that they happen afterwards.

Attachments:

* [gendef-doc-bug.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/610/610ca7806a103e5c423dfe7fb5aae9213ddd26db/1a8f6a4455152936d72a442f0769e5a428966582.patch) (581 bytes) - added by thijs on 2008-11-09 17:08:20Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3524 3524
type__defect defect
reporter__eggy eggy
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1225916258000000 1225916258000000
changetime__1226777359000000 1226777359000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @thijstriemstra|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Great, please apply.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [25352]) Apply gendef-doc-bug.patch: fix documentation bug by removing an incorrect statement about the largeFibonnaciNumber thread.

Author: thijs
Reviewer: exarkun
Fixes: [#3524](https://github.com/twisted/twisted/issues/3524)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
